src/noisepy/seis/plotting_modules.py

```diff
@@ -50,26 +50,28 @@ def plot_waveform(
 
     USAGE:
     -----------------------
-    plot_waveform(raw_data_store, '2023-10-21T00:00:00+0000 - 2023-10-21T12:00:00+0000', 'CI','BLC',0.01,0.5)
+    plot_waveform(raw_data_store, '2015-03-22T10:00:00+0000 - 2015-03-22T10:10:00+0000', 'CI','BLC',0.01,0.5)
```

Comment:
Here I meant that the argument needs to be a DateTimeRange, vs a string:
```suggestion
    plot_waveform(raw_data_store, DateTimeRange('2023-10-21T00:00:00+0000','2023-10-21T12:00:00+0000'), 'CI','BLC',0.01,0.5)
```

Reply:
Done